src/pages/index.tsx

```diff
@@ -2,144 +2,86 @@
 import { jsx, Box, Flex, Styled, Layout } from 'theme-ui';
 import SEO from "../components/seo"
 import { Link } from "../components/nav"
-import { Theme } from "../components/theme"
-import "@csstools/normalize.css"
+import GridLayout from '../components/layout';
+import { Fragment } from 'react';
 
-const padding = [3, 4, 5]
-const minorColumnWidth = ["48%", "38.196601125%"]
+const number = { m: 0, color: 'accent' }
 
 const IndexPage = () => (
-  <Theme>
```

Comment:
You can use the `<>` format for a fragment.

Reply:
This is not possible when using theme-ui unfortunately! https://github.com/system-ui/theme-ui/issues/195#issuecomment-513816495